### DISCUSSION TITLE
大佬，萌新还是不知道配置后该怎么用，能写份简单调用接口的demo吗？

### DISCUSSION THREAD

**Discussion by Umendboom** [nkxingxh/MiraiEz#7] [Q&A] [answered]
第一次搞这个，不知道哪一步出错了。如果有个简易的demo就好了。

**Comment by nkxingxh** [comment, marked as answer]
你看示例插件

**Comment by MR-XieXuan** [comment]
@Umendboom 可以根据Bilibili的视频https://www.bilibili.com/video/BV1QK411i7Zq/?spm_id_from=333.999.0.0进行配置
